script.js

```diff
@@ -1,7 +1,57 @@
 /* eslint-disable linebreak-style */
+const cardsData = [
+  {
+    id: 1,
+    title: 'Card 1',
+    image: './assets/images/card-image.png',
+    description:
+        "1 Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  },
```

Comment:
- [ ] Please your project object misses the following information: technologies, link to live version, and link to the project source code. 
### **The required object structure**
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```